server/src/routes/api/v1/destinationsReviewRouter.js

```diff
@@ -0,0 +1,25 @@
+import express from "express"
+import { Review, Destination } from "../../../models/index.js"
+import cleanUserInput from "../../../services/cleanUserInput.js"
+import objection from "objection"
+const { ValidationError } = objection
+
+const destinationReviewsRouter = new express.Router({ mergeParams: true})
+
+destinationReviewsRouter.post("/", async (req, res) => {
+    const formInput = cleanUserInput(req.body)
+    try { 
```

Comment:
here you will also need to get the signed in current user `req.user` and the merged destination `id` of the POST request before inserting to create the record